dns/dyndns/src/etc/inc/plugins.inc.d/dyndns/phpDynDNS.inc

```diff
@@ -1302,6 +1334,9 @@ class updatedns
             case 10:
                 $error = "Dynamic DNS ({$this->_dnsHost}): No change in my IP address and/or " . $this->_dnsMaxCacheAgeDays . " days has not passed. Not updating dynamic DNS entry.";
                 break;
+            case 11:
+                $status = 'Dynamic DNS: (ERROR!) No Token Provided.';
```

Comment:
The GUI is really restrictive in this regard.. the help label mentions "token" but ultimately the field label remains username so changing this can confuse users who don't see the help text.